src/System.Security.Cryptography.Cng/src/System.Security.Cryptography.Cng.builds

```diff
@@ -6,28 +6,12 @@
       <OSGroup>Windows_NT</OSGroup>
     </Project>
     <Project Include="System.Security.Cryptography.Cng.csproj">
-      <OSGroup>Windows_NT</OSGroup>
-      <TargetGroup>netstandard1.6</TargetGroup>
```

Comment:
@bartonjs are you aware if any bugfixes for those platforms are here and didn't make it to release/1.1.0 branch? I ask that because the plan is that any servicing for older frameworks should come from release branches instead of from master.